### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by AstraLuma** [Textualize/frogmouth#56] [Ideas]
So I'm thinking about using frogmouth as the suggested browser for in-repo docs.

Normally, I use sphinx+rst for docs hosted in the repo, but in this case, I don't need all of its formatting and organization features, and I think I want the text editor to be the primary interface, with the option for a nice browser.

I think frogmouth and a pile of markdown fits the bill?

**Comment by AstraLuma** [comment]
One thing that would add to it: Be able to specify the "root" of the docs, so the Local tab only shows the pertinent stuff. And possibly default to it.

**Comment by davep** [comment]
Thanks @AstraLuma; glad you're enjoying it.

While I'm not sure I'd want to be adding something to stop anyone navigating to anywhere (that'd touch a lot of things inside the code), be aware that you can pass a directory on the command line and the local files panel will default to showing that hierarchy. So:

```sh
$ frogmouth /some/location/for/my/docs
```

will have set the default location in the navigation panel.

**Reply by AstraLuma** [reply]
Oh nice! It would be cool to be able to combine that with a starting file, but yeah, that's pretty much exactly what I was thinking.
